TODO/static/JS/login-register_form_confirmation.js

```diff
@@ -0,0 +1,109 @@
+window.onload=function() {
+    var login_form = document.querySelector('#login-form.login-form');
+    var registration_form = document.querySelector('#registration-form.login-form');
+
+    if (login_form) {
+        login_form.addEventListener('submit', confirm_login_form);
+    }
+
+    if (registration_form) {
+        registration_form.addEventListener('submit', confirm_register_form);
+    }
+}
+
+
+function check_username_length(username) {
+    var result = false;
+    if (username.length < 15) {
+        result = true;
+    }
+
+    return result;
```

Comment:
```suggestion
    return username.length < 15;
```

Am I missing something?

Reply:
Fair enough)